cms/static/js/views/group_edit.js

```diff
@@ -1,10 +1,14 @@
+/**
+ * This class defines an edit view for groups within content experiment group configurations.
```

Comment:
I think this file (and class it defines) needs to be renamed. "group_edit" is way too generic. How about experiment_group_edit? I know we haven't used "experiment" for the other experiment files (because of the desire to avoid a massive rename), but group_configurations_group_edit.js is confusing and unwieldy.